domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java

```diff
@@ -353,6 +388,7 @@ public void testProcessQueryException() {
     );
   }
 
+  /* DISABLED
```

Comment:
Can you uncomment this test?

Reply:
Yes. Will do. Can not just be commented out, because uses Impl classes from old driver, so I'l have to find new ones or implement interfaces etc.